src/test/java/teammates/storage/sqlapi/FeedbackResponseCommentsDbTest.java

```diff
@@ -0,0 +1,141 @@
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.FeedbackParticipantType;
+import teammates.common.exception.EntityAlreadyExistsException;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.HibernateUtil;
+import teammates.storage.sqlentity.FeedbackResponseComment;
+import teammates.test.BaseTestCase;
+
+/**
+ * SUT: {@code FeedbackResponseCommentsDb}.
+ */
+
+public class FeedbackResponseCommentsDbTest extends BaseTestCase {
+
+    private static final Long TYPICAL_ID = 100L;
+    private FeedbackResponseCommentsDb feedbackResponseCommentsDb;
+    private MockedStatic<HibernateUtil> mockHibernateUtil;
+
+    @BeforeMethod
+    public void setUpMethod() {
+        mockHibernateUtil = mockStatic(HibernateUtil.class);
+        feedbackResponseCommentsDb = spy(FeedbackResponseCommentsDb.class);
+    }
+
+    @AfterMethod
+    public void teardownMethod() {
+        mockHibernateUtil.close();
+
+    }
+
+    @Test
+    public void testCreateComment_commentDoesNotExist_success()
+            throws InvalidParametersException, EntityAlreadyExistsException {
+        FeedbackResponseComment comment = getTypicalResponseComment();
+
+        feedbackResponseCommentsDb.createFeedbackResponseComment(comment);
+
+        mockHibernateUtil.verify(() -> HibernateUtil.persist(comment));
+    }
+
+    @Test
+    public void testCreateComment_commentAlreadyExists_throwsEntityAlreadyExistsException() {
+        FeedbackResponseComment comment = getTypicalResponseComment();
+
+        mockHibernateUtil.when(() -> HibernateUtil.get(FeedbackResponseComment.class, TYPICAL_ID)).thenReturn(comment);
+
+        EntityAlreadyExistsException ex = assertThrows(EntityAlreadyExistsException.class,
+                () -> feedbackResponseCommentsDb.createFeedbackResponseComment(comment));
+
+        assertEquals("Trying to create an entity that exists: " + comment.toString(), ex.getMessage());
+        mockHibernateUtil.verify(() -> HibernateUtil.persist(comment), never());
+    }
+
+    @Test
+    public void testGetComment_commentAlreadyExists_success() {
+        FeedbackResponseComment comment = getTypicalResponseComment();
+
+        mockHibernateUtil.when(() -> HibernateUtil.get(FeedbackResponseComment.class, TYPICAL_ID)).thenReturn(comment);
+
+        FeedbackResponseComment commentFetched = feedbackResponseCommentsDb.getFeedbackResponseComment(TYPICAL_ID);
+
+        mockHibernateUtil.when(() -> HibernateUtil.get(FeedbackResponseComment.class, TYPICAL_ID)).thenReturn(comment);
+        assertEquals(comment, commentFetched);
+    }
+
+    @Test
+    public void testGetComment_commentDoesNotExist_returnsNull() {
+        mockHibernateUtil.when(() -> HibernateUtil.get(FeedbackResponseComment.class, 101L)).thenReturn(null);
```

Comment:
Can we create a variable in this test for 101L or at the top (since it is also used later - line 113), to make it clear what it is

Reply:
Good catch!